modules/apps/cookies/cookies-test/src/test/java/com/liferay/cookies/internal/events/CookiesDomainTest.java

```diff
@@ -49,7 +60,8 @@ public void testDomain2() throws Exception {
 		mockHttpServletRequest.setServerName("www.liferay.com");
 
 		Assert.assertEquals(
-			StringPool.BLANK, CookieKeys.getDomain(mockHttpServletRequest));
+			".liferay.com",
+			CookiesManagerUtil.getDomain(mockHttpServletRequest));
```

Comment:
That's an interesting issue. These tests were in `CookieKeysTest` which was in `portal-impl` package. Somehow during the test run there, it defines a `catalina.base` environment variable with `.` as value. Because of this the following value will be true.
```	
private static final boolean _SESSION_COOKIE_USE_FULL_HOSTNAME =
	GetterUtil.getBoolean(
		PropsUtil.get(
			PropsKeys.SESSION_COOKIE_USE_FULL_HOSTNAME,
			new Filter(ServerDetector.getServerId())));
```
That's because the `ServerDetector.getServerId()` will think we are running in a Tomcat environment and the default value in `portal.properties` is:
```
session.cookie.use.full.hostname=false
session.cookie.use.full.hostname[tomcat]=true
```
However, in the `cookies` the test runner does not define the `catalina.base` or any other environment variable which will make the server unidentifiable and falls back to false.

I talked with Ray about it but I guess this `StringPool.BLANK` was just a necessary thing to make the test pass. The whole test is a bit unnecessary as we test both cases in `testDomain4` and `testDomain5`.